frontend/src/core/comments/components/Comment.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Comment from './Comment';
+
+
+describe('<Comment>', () => {
+    const DEFAULT_COMMENTS = {
+        author: '',
+        username: '',
+        userGravatarUrlSmall: '',
+        createdAt: '',
+        dateIso: '',
+        content: 'Comment for testing',
+        translation: 0,
+        id: 1,
+    }
+
+    describe('renderUser', () => {
```

Comment:
When testing a small component like this one you can skip this level of abstraction and put all your tests in the top-level `describe`.